source/visionEnhancementProviders/NVDAHighlighter.py

```diff
@@ -396,7 +400,10 @@ def _run(self):
 		window = self.window = self.customWindowClass(self)

```

Comment:
I think there is a chance that on disable/enable cycling, referring to `self.window` is problematic. In the `__init__` method `self.window` is set to None. If `__init__` gets called before `_run` exits the `while` loop then we wont call `self.window.destroy()`. Can you use non-instance variables after this point? I don't think `self.timer` is used outside this function, if so, then I'd prefer it also be a local variable.